nebula_ros/config/lidar/hesai/Pandar128E4X.param.yaml

```
@@ -0,0 +1,26 @@
+/**:
+  ros__parameters:
+    host_ip: "255.255.255.255"
+    sensor_ip: "192.168.1.201"
+    data_port: 2368
+    gnss_port: 10110
+    packet_mtu_size: 1500
+    launch_hw: true
+    setup_sensor: true
+    frame_id: hesai
+    diag_span: 1000
+    min_range: 0.3
+    max_range: 300.0
+    cloud_min_angle: 0 
+    cloud_max_angle: 360
+    scan_phase: 0.0
+    sensor_model: Pandar128E4X
+    calibration_file: $(find-pkg-share nebula_decoders)/calibration/hesai/$(var sensor_model).csv
+    rotation_speed: 600
+    return_mode: Dual
+    ptp_profile: "1588v2"
+    ptp_domain: 0
+    ptp_transport_type: UDP
```

OT128 only supports `L2` transport
```suggestion
    ptp_transport_type: L2
```